src/convnets/resnet.jl

```diff
@@ -135,17 +136,20 @@ See also [`resnet`](#).
 - `shortcut_config`: the type of shortcut style (either `:A`, `:B`, or `:C`)
 - `block`: a function with input `(inplanes, outplanes, downsample=false)` that returns
            a new residual block (see [`Metalhead.basicblock`](#) and [`Metalhead.bottleneck`](#))
+- `connection`: the binary function applied to the output of residual and skip paths in a block
 - `nclasses`: the number of output classes
 """
 struct ResNet
   layers
 end
 
-function ResNet(channel_config, block_config, shortcut_config; block, nclasses = 1000)
+function ResNet(channel_config, block_config, shortcut_config;
+                block, connection = (x, y) -> @. relu(x + y), nclasses = 1000)
```

Comment:
Good opportunity to extract `connection` as an internal helper function so that it (de)serializes better?

Reply:
`addrelu` and `reluadd`?